docs/source/migration.mdx

```diff
@@ -72,68 +202,60 @@ Apollo Server 4 removes the below integration packages:
 
-> We are looking for volunteers to help [build](/integrations/building-integrations) and maintain these integrations!  [Join the discussion](https://github.com/apollographql/apollo-server/labels/integration-collaborators) about your favorite web framework today.
+Additionally, in Apollo Server 3 the `apollo-server-express` package supported both Express and its older predecessor [Connect](https://github.com/senchalabs/connect) using the same code. In Apollo Server 4, `expressMiddleware` no longer supports Connect. An interested developer could [build a Connect-specific middleware](/integrations/building-integrations) using the Apollo Server 4 API. A PR to this migration guide is welcome if someone does this!
 
+### Other packages merged into `@apollo/server`
 
-### `body-parser` & `cors`
+As described above, the functionality provided by the `apollo-server-core`, `apollo-server`, and `apollo-server-express` packages in AS3 has been combined into the single `@apollo/server` package in AS4.
 
-Apollo Server 4 no longer provides built-in support for the [`cors`](https://github.com/expressjs/cors) or [`body-parser`](https://www.npmjs.com/package/body-parser) libraries (excluding the [`startStandaloneServer`](#startstandaloneserver) function).
+But wait: there's more! The following packages have *also* been combined into `@apollo/server`:
+- [`apollo-server-errors`](https://www.npmjs.com/package/apollo-server-errors)
+- [`apollo-server-plugin-base`](https://www.npmjs.com/package/apollo-server-plugin-base)
+- [`apollo-server-types`](https://www.npmjs.com/package/apollo-server-types)
 
-If you want to use either `cors` or `body-parser`, you can manually add both libraries to your framework-specific middleware.
 
-Below is an example of setting up `cors` and `body-parser` using the [`expressMiddleware`](#expressmiddleware) function:
+### Other renamed packages
 
-<MultiCodeBlock>
+The `apollo-server-plugin-response-cache` and `apollo-server-plugin-operation-registry` plugin packages have been renamed to `@apollo/server-plugin-response-cache` and `@apollo/server-plugin-operation-registry` respectively.
 
-```ts
-import { ApolloServer, ApolloServerPluginDrainHttpServer } from '@apollo/server';
-import { expressMiddleware } from '@apollo/server/express4';
-import express from 'express';
-import http from 'http';
-import cors from 'cors';
-import bodyParser from 'body-parser';
+The [`apollo-reporting-protobuf`](https://www.npmjs.com/package/apollo-reporting-protobuf) (which is an internal implementation detail of the usage reporting plugin) has been renamed to `@apollo/usage-reporting-protobuf`.
 
-// Set up Express
-const app = express();
-const httpServer = http.createServer(app);
+Note that once AS4 is released, all actively maintained Apollo packages will start with `@apollo/`. This leaves the `apollo-` namespace open for community integration packages (e.g., `apollo-server-fastify`).
```

Comment:
Isn't it explicitly _not_ the plan to hand over publishing rights to these packages (`apollo-server-fastify` is currently ours)? I'm assuming this is a mistake, but if not I'd push back pretty hard against it at least for security reasons.

Reply:
Good question. I think we do want to leave `apollo-` open, but we should probably have a new scheme for as4 package names, which frankly will also just make it easier to see "oh you're combining two things that don't go together" issues.

Reply:
eg `apollo-server-integration-fastify` or `apollo-server-middleware-fastify` or something.

Reply:
Changing to `apollo-server-integration-fastify`